### PR TITLE
added moveit yaml files to cob_hardware_config

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

**Comment by NadiaHG** [comment]
Requires https://github.com/ipa320/cob_robots/pull/518

**Comment by NadiaHG** [comment]
Have to be tested on the robot 
@ipa-mdl what do you think about this solution?

File: cob_moveit_config/launch/planning_context.xml [hunk 1]
```diff
@@ -2,32 +2,16 @@
...
   <arg name="robot_description" default="robot_description"/>
 
   <!-- Load universal robotic description format (URDF) -->
   <include file="$(find cob_hardware_config)/common/upload_robot.launch" if="$(arg load_robot_description)">
     <arg name="robot" value="$(arg robot)" />
+    <arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)" />
+    <arg name="planning_context" value="true" />
   </include>
 
-  <!-- The semantic description that corresponds to the URDF -->
```
Comment (mathias-luedtke) [inline cob_moveit_config/launch/planning_context.xml:15]: We need the other parameters, too!

**Comment by mathias-luedtke** [comment]
I don't think this will work.
It's  fine for collision_monitor, but the planner will be broken without the parametrs

**Comment by NadiaHG** [comment]
@ipa-mdl which parameters are missed?
See https://github.com/ipa-nhg/cob_robots/blob/03cfccfa4c3e4bd6a2e71071ff57f8c78b7d48de/cob_hardware_config/common/upload_robot.launch

**Comment by mathias-luedtke** [comment]
Sry, I missed the changes in cob_robots.

So now cob_moveit_config is distributed between cob_manipulation and cob_hardware_config.
I don't like this for maintenance reasons and dependency-wise.

Perhaps we should move the config to cob_hardware_config. (or as cob_moveit_config in cob_robots) and move the launch files into a cob_moveit_bringup (?) package.

**Comment by fmessmer** [comment]
I really don't understand why yoj had such a problem with cob_bringup having a dependency to cob_manipulation and this moveit...
as we now use moveit not only for planning but also for collision checking it became an essential part of bringup (despite me considering planning essential anyway)!
Having all configs and launch in a single package was way better to maintain...though it still required some knowledge...so ease only restructure in case you knkw what you are doing..
I

**Comment by NadiaHG** [comment]
As @ipa-mdl suggested I renamed the `cob_moveit_config` to `cob_moveit_bringup` and moved all the config files to `cob_hardware_config` https://github.com/ipa320/cob_robots/pull/518/commits/ebade00fc55fe34421a1ad7505dac514e7becb51

**Comment by fmessmer** [comment]
closing as it is handled in related PRs https://github.com/ipa320/cob_manipulation/pull/88 https://github.com/ipa320/cob_manipulation/pull/91 as well as https://github.com/ipa320/cob_robots/pull/573 https://github.com/ipa320/cob_robots/pull/567

**Pull request closed without merging**